Ekwipunek.py

```diff
@@ -118,12 +121,12 @@ def __init__(
         wartosc: int,
         ilosc: int,
         mana: int,
-        dzialanie: dict,
+        dzialanie: dict, # typing, opisz co znaczy działanie
         krag: int = None,
     ):
         super().__init__(nazwa, wartosc, ilosc)
         self._mana = mana
-        self._dzialanie = dzialanie
+        self._dzialanie = dzialanie # Pole działanie powinno byc dla innych klas również
```

Comment:
Co masz na mysli? W tym konkretnym wypadku tyczy sie to tego, ze magia ma rozne zastosowania, takie jak zadawanie obrazen czy tez leczenie. Z kolei bron, pancerz etc. tego nie ma, chyba, ze czegos nie widze. Jedynie gdzie to widze to w 'Pozostale'